test/e2e/support/commands.js

```diff
@@ -703,37 +703,100 @@ Cypress.Commands.add("expectCredentialsSectionEmpty", () => {
 });
 
 // deletePCCContent
-// Purpose: Minimal PCC cleanup using in-memory values captured during the test.
+// Purpose: Delete ALL PCC content for the test account to ensure clean state.
 // - Requires Cypress.env("PCC_ACCESS_TOKEN") set by setPCCCredential()
-// - Requires Cypress.env("LAST_PCC_CONTENT_ID") set by the test after reading direct_url
+// - Requires account_id from pccConfig
 Cypress.Commands.add("deletePCCContent", () => {
-  const contentId = Cypress.env("LAST_PCC_CONTENT_ID");
   const token = Cypress.env("PCC_ACCESS_TOKEN");
   const env = Cypress.env("CONNECT_CLOUD_ENV") || "staging";
+  const pccConfig = Cypress.env("pccConfig");
+  const accountId = pccConfig?.pcc_user_ccqa3?.account_id;
```

Comment:
Can we store the `accountId` in a similar way to `PCC_ACCESS_TOKEN`? This looks pretty different, but I'm not entirely sure what `pccConfig` looks like so figured I'd ask.